_data/sidebars/docs_sidebar.yml

```diff
@@ -241,6 +241,10 @@ entries:
       url: /tooling-rbf-shape.html
       output: web, pdf
 
+    - title: Micro-Manager
```

Comment:
```suggestion
    - title: Micro Manager
```

maybe even:

```suggestion
    - title: Micro manager
```

Reply:
So far we have stuck to **Micro Manager** as the name of the software. I would suggest sticking to that here too.